apps/01PigLatin.js

```diff
@@ -5,13 +5,47 @@ var prompt = require('prompt');
 prompt.start();
 
 
-function pigLatin(word) {
 
+function pigLatin(word) {
     // Your code here
+    var word = 'CRaz';
```

Comment:
Should you be defining `var word = 'CRaz';` here?

Might that be interfering with something else?